enterprise/server/cmd/ci_runner/main.go

```diff
@@ -697,6 +703,39 @@ func ensurePath() error {
 	return os.Setenv("PATH", "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin")
 }
 
+func getBazeliskCommand() (string, error) {
+	wd, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+	// If bazelisk is already in the workspace root, use that.
+	if _, err := os.Stat(bazeliskBinaryName); err != nil {
+		if !os.IsNotExist(err) {
+			return "", err
+		}
+	} else {
+		return filepath.Join(wd, bazeliskBinaryName), nil
+	}
```

Comment:
I would ignore this error and fall through to your case below. It also would make this code a little easier to follow:

	if _, err := os.Stat(bazeliskBinaryName); err == nil {
        return filepath.Join(wd, bazeliskBinaryName), nil
	}